org.eclipse.jdt.ui/META-INF/MANIFEST.MF

```diff
@@ -102,7 +102,6 @@ Require-Bundle:
  org.eclipse.core.resources;bundle-version="[3.14.0,4.0.0)",
  org.eclipse.core.variables;bundle-version="[3.2.200,4.0.0)",
  org.eclipse.jdt.core;bundle-version="[3.34.0,4.0.0)",
```

Comment:
I assume Stephan added that because of the too old version used in jdt.core `[3.34.0,4.0.0)`.
Please update `[3.34.0,4.0.0)` to `[3.36.0,4.0.0)`

Reply:
OK.